lib/ClangImporter/ImportDecl.cpp

```diff
@@ -5217,11 +5218,14 @@ SwiftDeclConverter::importAsOptionSetType(DeclContext *dc, Identifier name,
       decl, AccessLevel::Public, Loc, name, Loc, None, nullptr, dc);
   structDecl->computeType();
 
-  ProtocolDecl *protocols[] = {cxt.getProtocol(KnownProtocolKind::OptionSet)};
+  ProtocolDecl *protocols[] = {ctx.getProtocol(KnownProtocolKind::OptionSet)};
   makeStructRawValued(Impl, structDecl, underlyingType,
                       {KnownProtocolKind::OptionSet}, protocols);
-  addSynthesizedTypealias(structDecl, cxt.Id_Element,
-                          structDecl->getDeclaredInterfaceType());
+  auto selfType = structDecl->getDeclaredInterfaceType();
+  addSynthesizedTypealias(structDecl, ctx.Id_Element,
+                          selfType);
+  addSynthesizedTypealias(structDecl, ctx.getIdentifier("ArrayLiteralElement"),
```

Comment:
Probably better to add this to KnownIdentifiers.

Reply:
I did this in a subsequent PR.